FabricExample/App.tsx

```diff
@@ -1,276 +1,72 @@
-  },
-  {
-    name: 'BokehExample',
-    icon: '✨',
-    title: 'Bokeh',
-    component: BokehExample,
-  },
-  {
-    name: 'BubblesExample',
-    icon: '🫧',
-    title: 'Bubbles',
-    component: BubblesExample,
-  },
-  {
-    name: 'ColorExample',
-    icon: '🌈',
-    title: 'Colors',
-    component: ColorExample,
-  },
-  {
-    name: 'ScreenStackHeaderConfigBackgroundColorExample',
-    icon: '🎨',
-    title: 'Screen header background color',
-    component: ScreenStackHeaderConfigBackgroundColorExample,
-  },
-  {
-    name: 'ScreenStackExample',
-    icon: '🥞',
-    title: 'Screen stack',
-    component: ScreenStackExample,
-  },
-  {
-    name: 'GestureHandlerExample',
-    icon: '👌',
-    title: 'Draggable circle',
-    component: GestureHandlerExample,
-  },
-  {
-    name: 'SvgExample',
-    icon: '🟢',
-    title: 'Animated SVG circle',
-    component: SvgExample,
-  },
-  {
-    name: 'BouncingBoxExample',
-    icon: '📦',
-    title: 'Bouncing box',
-    component: BouncingBoxExample,
-  },
-  {
-    name: 'AnimatedSensorExample',
-    icon: '📡',
-    title: 'useAnimatedSensor',
-    component: AnimatedSensorExample,
-  },
-  {
-    name: 'FrameCallbackExample',
-    icon: '🗣',
-    title: 'useFrameCallback',
-    component: FrameCallbackExample,
-  },
-  {
-    name: 'ScrollViewExample',
-    icon: '📜',
-    title: 'useAnimatedScrollHandler',
-    component: ScrollViewExample,
-  },
-  {
-    name: 'ScrollToExample',
-    icon: '🦘',
-    title: 'scrollTo',
-    component: ScrollToExample,
-  },
-  {
-    name: 'ScrollViewPositionExample',
-    icon: '𝌍',
-    title: 'useScrollViewPosition',
-    component: ScrollViewOffsetExample,
-  },
-  {
-    name: 'ArticleProgressExample',
-    icon: '📰',
-    title: 'Article Progress Example',
-    component: ArticleProgressExample,
-  },
-  {
-    name: 'MeasureExample',
-    icon: '📐',
-    title: 'measure',
-    component: MeasureExample,
-  },
-  {
-    name: 'WorkletExample',
-    icon: '🧵',
-    title: 'runOnJS / runOnUI',
-    component: WorkletExample,
-  },
-  {
-    name: 'TransformExample',
-    icon: '🔄',
-    title: 'Transform',
-    component: TransformExample,
-  },
-  {
-    name: 'WidthExample',
-    icon: '🌲',
-    title: 'Layout props',
-    component: WidthExample,
-  },
-  {
-    name: 'RefExample',
-    icon: '🦑',
-    title: 'forwardRef & useImperativeHandle',
-    component: RefExample,
-  },
-  {
-    name: 'ChessboardExample',
-    icon: '♟️',
-    title: 'Chessboard',
-    component: ChessboardExample,
-  },
-  {
-    name: 'OverlappingBoxesExample',
-    icon: '🔝',
-    title: 'z-index & elevation',
-    component: OverlappingBoxesExample,
-  },
-  {
-    name: 'NewestShadowNodesRegistryRemoveExample',
-    icon: '🌓',
-    title: 'Conditional',
-    component: NewestShadowNodesRegistryRemoveExample,
-  },
-  {
-    name: 'EmptyExample',
-    icon: '👻',
-    title: 'Empty',
-    component: EmptyExample,
-  },
-];
+export default function WidthExample() {
```

Comment:
I think that we need to revert changes in this file also

Reply:
Done in ac5be7dbdd77a9c91a3ebefb8e22a7590e88b2bc